lib/common.py

```diff
@@ -455,38 +473,48 @@ def shutdown(self, force=False, verify=False):
     def try_get_and_store_ip(self):
         ip = self.param_get('networks', '0/ip', accept_unknown_key=True)
 
-        if not ip:
+        if not ip or ip.startswith('169.254.'):
```

Comment:
I'd add a comment about this fake IP so we remember in the future.

Reply:
Done